web/index.html

```diff
@@ -0,0 +1,234 @@
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Vange-rs: WebAssembly + Rust</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="assets/road.css" rel="stylesheet">
+</head>
+
+<body class="h-full">
+    <div id="controls" class="absolute py-2 px-4 flex flex-col font-mono h-full">
+        <H1 class="font-bold text-lg my-4">Vange-rs: WebAssembly + Rust</H1>
+        <p class="font-bold my-2">Controls (click into canvas to obtains keyboard focus)</p>
+        <p>Move: WASD</p>
+        <p>Jump: Left Alt</p>
+        <p>Turbo: Left Shift</p>
+        <p>Reset: R</p>
+        <p>Car view: P</p>
+
+
+        <p class="font-bold mt-8 mb-2">Select world and press GO! Then wait patiently...</p>
+
+
+        <div class="flex flex-row items-center">
+            <div>World:</div>
+            <select id="level" class="mx-4 p-2" onchange="javascript:updateSettings()">
+                <option>Ark-a-Znoy</option>
+                <option>Boozeena</option>
+                <option>hMok</option>
+                <option>Khox</option>
+                <option selected>Threall</option>
+                <option>Weexow</option>
+                <option>Xplo</option>
+            </select>
+            <button onclick="javascript:go()"
+                class="border px-2 py-1 border-gray-400 rounded bg-gradient-to-r from-yellow-400 via-red-500 to-pink-500">
+                GO!
+            </button>
+        </div>
+
+        <textarea id="settings" class="flex-grow my-2">
+        </textarea>
+    </div>
+    <div class="flex flex-col h-full w-full bg-gray-200 justify-center overflow-hidden">
+        <canvas id="canvas" data-raw-handle="1" width="1280" height="720" class="w-full"
+            style="height: 56vw; display: none;"></canvas>
+    </div>
+
+    <script type="module">
+        import init from "./road.js";
+
+        async function go() {
+            window.fs = await fsPromise;
+
+            document.getElementById("controls").style.display = "none";
+
+            const canvas = document.getElementById("canvas");
+            canvas.addEventListener("webglcontextlost", function (e) { alert('WebGL context lost. You will need to reload the page.'); e.preventDefault(); }, false);
+
+            var gl = canvas.getContext('webgl2', {
+                alpha: false,
+                antialias: false,
+                depth: false,
+                majorVersion: 2,
+                minorVersion: 0,
+                stencil: false,
+            });
+            canvas.style.display = "block";
+
+            init().then(async (m) => { }).catch(console.error);
+        }
+
+        window.go = go;
+    </script>
+
+    <script>
+        const filesToFetch = [
```

Comment:
would be nice to have this queried instead of hard-coded, but we can do it as a follow-up